client/components/input/input.css

```diff
@@ -1,3 +1,117 @@
+
+input, textarea {
+  outline: none;
+  border: none;
+  font-weight: inherit;
+  font-size: inherit;
+  line-height: inherit;
+  color: var(--primary-dark);
+  caret-color: var(--primary-blue);
+  letter-spacing: inherit;
+  background: var(--primary-white);
+  max-width: 475px;
+} 
+textarea {
+  font-weight: 300;
 }
+
+.input::placeholder {
+font-weight: inherit;
+font-size: inherit;
+line-height: inherit;
+color: var(--placeholder);
+}
+input:focus::placeholder {
+  color: transparent;
+}
+
+input[type="number"]::-webkit-outer-spin-button,
+input[type="number"]::-webkit-inner-spin-button {
+  -webkit-appearance: none;
+  margin: 0;
+}
+input[type="number"]{
+  -moz-appearance: textfield;
+}
+
+fieldset {
+   display: flex;
+   flex-direction: column-reverse;
+   margin-bottom: 15px;
+    border: 1px solid var(--border-unfocused);
+    border-radius: 8px;
+    outline: none;
+    max-width: 546px;
+    max-height: 80px;
+    background: var(--primary-white);
+}
+.textArea {
+  max-width: 546px;
+  max-height: 216px; 
+  min-height:216px;
+  }
+
+fieldset:valid {
+background: var(--primary-white);
+background-image: url("data:image/svg+xml,%3Csvg width='22' height='18' viewBox='0 0 22 18' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill-rule='evenodd' clip-rule='evenodd' d='M21.6585 0.247422C22.0741 0.611104 22.1163 1.24287 21.7526 1.6585L7.75258 17.6585C7.57032 17.8668 7.30989 17.9902 7.03328 17.9994C6.75666 18.0087 6.4886 17.9028 6.29289 17.7071L0.292893 11.7071C-0.0976311 11.3166 -0.0976311 10.6834 0.292893 10.2929C0.683417 9.90237 1.31658 9.90237 1.70711 10.2929L6.95129 15.5371L20.2474 0.341494C20.6111 -0.0741428 21.2429 -0.11626 21.6585 0.247422Z' fill='%230061FF'/%3E%3C/svg%3E"); 
+background-repeat: no-repeat;
+background-position: 95% 50%;
+width:100%;
+}
+
+fieldset:user-invalid {
+background: var(--primary-white);
+border: 1px solid var(--invalid-red);
+background-image: url("data:image/svg+xml,%3Csvg width='26' height='23.78' viewBox='0 0 20 23.' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M25.0669 20.9425L14.415 1.16125C13.66 -0.24125 11.6487 -0.24125 10.8931 1.16125L0.241848 20.9425H0.241848C-0.281788 21.9151 0.0821392 23.128 1.0547 23.6516C1.34525 23.808 1.66999 23.8902 1.99997 23.8906H23.3056C24.4102 23.8908 25.3057 22.9955 25.3059 21.8909C25.3059 21.5599 25.2238 21.234 25.0669 20.9425V20.9425ZM12.6544 20.8281C11.964 20.8281 11.4043 20.2685 11.4043 19.5781C11.4043 18.8878 11.964 18.3281 12.6544 18.3281C13.3447 18.3281 13.9044 18.8878 13.9044 19.5781C13.9044 20.2685 13.3447 20.8281 12.6544 20.8281ZM14.0119 8.25625L13.6531 15.8813C13.6531 16.4335 13.2054 16.8813 12.6531 16.8813C12.1008 16.8813 11.6531 16.4335 11.6531 15.8813L11.2943 8.25937V8.25937C11.2612 7.51004 11.8417 6.87569 12.5911 6.84252C12.6068 6.84182 12.6224 6.8414 12.6381 6.84125H12.6512C13.4016 6.84086 14.0103 7.44888 14.0107 8.1993C14.0107 8.21954 14.0102 8.23978 14.0094 8.26L14.0119 8.25625Z' fill='%23B30038'/%3E%3C/svg%3E%0A"); 
```

Comment:
On my opinion, it is better to save image to file in some special directory and set background-image property as url to file.
In this case this image could be used by other team in other component